sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java

```diff
@@ -210,6 +212,7 @@ public void gatewayDiagnostics() throws Exception {
             assertThat(diagnostics).containsPattern("(?s).*?\"activityId\":\"[^\\s\"]+\".*");
             assertThat(createResponse.getDiagnostics().getDuration()).isNotNull();
             assertThat(createResponse.getDiagnostics().getContactedRegionNames()).isNotNull();
+            assertThat(diagnostics).contains("\"readRequestsFallbackEnabled\":\"true\"");
```

Comment:
in order for it to be shown in the cosmos diagnostics, we need to wire up in **DiagnosticsClientConfig** in  **RxDocumentClientImpl.java**

```
   this.diagnosticsClientConfig = new DiagnosticsClientConfig();
   this.diagnosticsClientConfig.withClientId(this.clientId);
   this.diagnosticsClientConfig.withActiveClientCounter(activeClientsCnt);
```